UIImageColors/Sources/UIImageColors.swift

```diff
@@ -247,7 +247,7 @@ extension UIImage {
         let imageColors = NSCountedSet(capacity: width*height)
         for x in 0..<width {
             for y in 0..<height {
-                let pixel: Int = ((width * y) + x) * 4
+                let pixel: Int = y * cgImage.bytesPerRow + x * 4
```

Comment:
Let's reintroduce the parenthesis: `((cgImage.bytesPerRow * y) + x) * 4`. It's easier to read

Reply:
It has to be without parenthesis.
y * cgImage.bytesPerRow is the start address of the current line, that added with x * 4 is the start address of the current pixel.

Reply:
Ah yes, you're right. Can we still include parentheses: `(cgImage.bytesPerRow * y) + (4 * x)`?